inst/extdata/child/intro-emissions_profile.Rmd

```diff
@@ -1,35 +1,41 @@
-The "emissions profile" measures the transition risk on the product-level of a
-company. The indicator is expressed in the percentage of all products that are
-at high risk, medium risk or low risk due to the products' relative greenhouse
-gas (GHG) emissions. The assessment is first performed on a product-level and
-can then be aggregated to the company-level.
+The "emissions profile" measures the absolute GHG emissions of a product in
+comparison to a chosen benchmark of products. The assessment is first performed
+on a product-level and can then be aggregated to the company-level. The profile
+is expressed as the share of a company's products that are in a "high",
+"medium", or "low" emission profile category, based on the comparison to the
+benchmark. A higher emission profile indicates a larger impact on climate change
+compared to the benchmark and can therefore also be interpreted as one climate
+risk component to assess products, companies, or loan portfolios.
 
-The "emissions profile" measures the relative GHG emissions per product. As a
-default option, each product is compared to the GHG emissions of every other
-product. Products with higher GHG emissions are considered to have a higher
-emission profile and hence also face a higher climate risk.
+As a first step to calculate the "emission profile", the relative GHG emissions
+per product are collected from a Life-Cycle-Analysis (LCA) database by matching
+the products from our company dataset to the products from the LCA dataset. As a
+second step, all products are ranked according to their GHG emissions. Thirdly,
+the products are grouped by the following 6 benchmarks:
 
-After identifying the GHG emissions for each product, the products are ranked
-according to their carbon footprint. The ranking method is explained in more
-detail in the
-[Thresholds](https://2degreesinvesting.github.io/tiltIndicator/articles/tiltIndicator.html#thresholds)
-section. In short, products with relatively low GHG emissions are assigned to
-the category "low", products with relatively high GHG emissions to the category
-"high" and the products in between to the category "medium".
+* "all": All products.
```

Comment:
Small style changes for consistency with the style I use elsewhere:

*  Changed bullets to `*`.
* Follow a colon with a full grammatical sentence -- starting with a capital letter and ending with ".".
* Escaped underscore with `\` to avoid it to be interpreted as the beginning if _itallics_.